### DISCUSSION TITLE
Using the validation outside of an adonis project

### DISCUSSION THREAD

**Discussion by e-e** [adonisjs/core#3593] [Help]
Is this supported? I tried simply installing `@adonisjs/validator`, but I get `node_modules/@adonisjs/validator/build/adonis-typings/index.d.ts' is not a module.` Using `require` also doesn't work. `console.log`ing the imported value gives:
```
{ default: [class ValidationProvider] { needsApplication: true } }
```
so, is it a configuration thing?

**Comment by aikrom** [comment]
I think this is possible, but you need to initialize/configure the validator instance yourself, and not use a provider, because you will not have a `@adonis/fold` - IoC in your project. But along with `@adonisjs/validator`, you will also have to install a extra `@adonisjs/*` packages and types into your project.
